init.lua

```diff
@@ -288,18 +309,18 @@ local function getRule(item)
         if loot.LootQuest then
             --look to see if Quantity attached to Quest|qty
             local _, position = string.find(lootData[firstLetter][itemName], '|')
-            if position then qKeep = tonumber(string.sub(lootData[firstLetter][itemName], position + 1)) else qKeep = 0 end
```

Comment:
any particular reason you swapped 1 tonumber here for a tostring and tonumber later?

Reply:
i was cleaning up warnings in vscode.